layouts/partials/docs.html

```diff
@@ -0,0 +1,24 @@
+<section class="section is-medium" style="margin: 0;">
+  <div class="container docs">
+    <div class="admonition">
+      <h3 class="title-2">API Reference</h3>
+      <p>Access comprehensive developer documentation for PyTorch-Ignite</p>
+      <a href="https://pytorch.org/ignite/engine.html">View API Reference →</a>
+    </div>
+    <div class="admonition">
+      <h3 class="title-2">Tutorials</h3>
+      <p>Get in-depth tutorials for beginners and advanced developers</p>
+      <a href="/tutorials/">View Tutorials →</a>
+    </div>
+    <div class="admonition">
+      <h3 class="title-2">How-to-Guides</h3>
+      <p>Get byte-size ready how-to short tips</p>
+      <a href="/how-to-guides/">View How-to-Guides →</a>
+    </div>
+    <div class="admonition">
+      <h3 class="title-2">Another one is what?</h3>
```

Comment:
Developer Guide = Contributing.md or Community page ?

Reply:
Then I think having 3 items is enough here as there is a proper contributing.md in ignite repo and community page is not that important to visit on the first visit to the website.

Reply:
OK, let's keep 3 of them